src/services/blockchain/ethereum/ethereum-blockchain-service.ts

```diff
@@ -76,14 +76,23 @@ export default class EthereumBlockchainService implements BlockchainService {
     } else {
       const gasPriceEstimate = await this.provider.getGasPrice();
       // Add 10% extra to gas price for each subsequent attempt
-      txData.gasPrice = gasPriceEstimate.add(gasPriceEstimate.mul(attempt * .01));
+      txData.gasPrice = EthereumBlockchainService.increaseGasPrice(gasPriceEstimate, attempt)
       logger.debug('Estimated Gas price: ' + txData.gasPrice.toString());
```

Comment:
Can we add the denomination in this comment too? "wei"

Reply:
done